packages/auth/amplify_auth_cognito/lib/src/auth_plugin_impl.dart

```diff
@@ -30,9 +30,11 @@ import 'package:flutter/services.dart';
 /// {@endtemplate}
 class AmplifyAuthCognito extends AmplifyAuthCognitoDart {
   /// {@macro amplify_auth_cognito.amplify_auth_cognito}
-  AmplifyAuthCognito()
+  AmplifyAuthCognito({
```

Comment:
We should add a doc comment to the constructor to callout that `credentialStorage` defaults to `AmplifySecureStorage` with a scope of "auth" and that changing the credential storage, or the credential storage config, will likely result in end users having to re-authenticate.

Something along the lines of:
```dart
  /// To change the default behavior of credential storage,
  /// provide a [credentialStorage] value. If no value is provided,
  /// [AmplifySecureStorage] will be used with a `scope` of "auth".
  ///
  /// **Warning:** Changing the credential provider, or customizing 
  /// the config value for [AmplifySecureStorage], will likely result in 
  /// end users having to re-authenticate then next time the app is opened.
```